test/common/common/utility_fuzz_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+
+  uint64_t out;
+  std::string stringBuffer(*buf, len);
```

Comment:
You need to use `string_buffer`, not `stringBuffer` for locals in Envoy style, please take a look at the style guide at https://github.com/envoyproxy/envoy/blob/master/STYLE.md if not familiar.